tracker/tracker/src/main/modules/network.ts

```diff
@@ -208,6 +217,10 @@ export default function (app: App, opts: Partial<Options> = {}) {
             return
           }
 
+          const isCached =
```

Comment:
We can use `transferedSize` for that purpose - this is at least [mentioned](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceResourceTiming#typical_resource_timing_metrics) in the MDN, but also in the [specification](https://w3c.github.io/resource-timing/#sec-performanceresourcetiming) (see "transferSize getter" section there). 
Specification also says, that the header size is taken as a const, because of some security issue. At the same time, `headerSize` is calculated from `transferedSize`, while the last one contains more information by itself (when it is 0, for example), so we could replace the first field with the second one (without replacing the full message, though with rewriting logic on backend).

Moreover, the statusCode is actually becomes available through the [WebAPI](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceResourceTiming/responseStatus) in the latest versions of browsers. Thus, we need to take advantage of this.